static/js/src/entry/user-portal/constants.js

```diff
@@ -0,0 +1,11 @@
+export const LOGIN_REDIRECT_KEY = 'postLoginRedirect';
+export const LOGGED_IN_FLAG_KEY = 'isLoggedIn';
+export const NON_STAFF_CONNECTION = 'Username-Password-Authentication';
+export const AUTH_DOMAIN = process.env.AUTH0_DOMAIN;
```

Comment:
Might be helpful to [document](https://github.com/texastribune/donations#environment) these for easy troubleshooting down the line. Mine barked at me in my console so it was pretty clear, but might help for someone setting up a new env.

Reply:
I shall!